app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/CreateServiceRequestController.php

```diff
@@ -108,9 +109,12 @@ public function store(
         DB::beginTransaction();
 
         try {
+            $serviceRequestStatus = ServiceRequestStatus::orderBy('id', 'asc')->firstOrFail();
             $serviceRequest = new ServiceRequest([
                 'title' => $data->pull('Main.title'),
                 'close_details' => $data->pull('Main.description'),
+                'status_id' => $serviceRequestStatus ? $serviceRequestStatus->id : null,
+                'status_updated_at' => $serviceRequestStatus ? now() : null,
```

Comment:
The null checks are not needed here because of the firstOrFail

Reply:
Checks removed.